static/usr/lib/udev/rules.d/90-rtc-sys-time-init.rules

```diff
@@ -0,0 +1,3 @@
+# When hctosys device rtc* is added create device unit dev-rtc*.
+# This is used by rtc-sys-time-init.service.
+ACTION=="add", SUBSYSTEM=="rtc", KERNEL=="rtc*", ATTR{hctosys}=="1", TAG+="systemd"
```

Comment:
I think there is something wrong here. We have 2 features. One to rollback wrong update from `RTC_HCTOSYS_DEVICE`. This has to be always enabled. The other is to emulate hctosys for some rtc devices not configurated as `RTC_HCTOSYS_DEVICE`. This has to be enabled conditionally, and it should not be expected `/dev/rtc` (because not `ATTR{hctosys}`).

Reply:
Thanks for pointing this out. Indeed hctosys=1 only happens when driver actually sets system time, which is not the case in older kernel with the issue! Now addressed.